test/test_generate/test-autograder/autograder-correct/environment.yml

```diff
@@ -1,10 +1,12 @@
-name: otter-env
 channels:
-    - defaults
-    - conda-forge
+- conda-forge
+- defaults
 dependencies:
-    - python=3.7
-    - pip
-    - nb_conda_kernels
-    - pip:
-        - -r file:requirements.txt
\ No newline at end of file
+- python>=3.6
+- r-base
+- r-essentials
+- r-devtools
+- pip
+- pip:
+  - -r requirements.txt
```

Comment:
do you know why this line no longer says "file:requirements.txt"? i think this might cause an error

Reply:
I am not sure actually... the custom-env one has "file:". Will take a closer look